### ISSUE TITLE
Create an affordance to track whether a project is alpha/beta/live

### ISSUE THREAD

**Issue by nkkl** [18F/a11y-metrics#13]
Reason being that a11y failures on a live site are significantly worse than in alpha.

**Comment by toolness** [comment]
Hmm, so one automated heuristic we can use for this is whether the site's homepage is on a subdomain of `app.cloud.gov`, like [`crime-data-explorer`](https://github.com/18F/crime-data-explorer) currently is.  If this heuristic is too inaccurate though, we can always have a separate hand-written YAML file that the dashboard uses to tell whether something is alpha, beta, or live.

**Comment by nkkl** [comment]
Let's hold on this until we can get it implemented properly. I don't think this is time-sensitive enough to warrant devoting energy to a workaround.
